weave/config.py

```diff
@@ -98,8 +98,16 @@ def get_index_column_names():
 
 def get_file_system():
     """Get the filesystem to be used for storing baskets."""
+    # return s3fs.S3FileSystem(
```

Comment:
Delete this if its not being used anymore

Reply:
Thanks, it's removed